### PR TITLE
Add filter in gradebook and students page to list only Students

### PR DESCRIPTION
**Pull request by jonatanschroeder**
I'm open to suggestions about tweaks in UI (e.g., different button order, different title).

### CODE REVIEW

File: pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs [hunk 1]
```diff
@@ -3,6 +3,20 @@ var bsTable;
 $(function() {
 
     bsTable = $('#usersTable').bootstrapTable({
+        buttons: {
+            studentsOnly: {
```
Comment (mwest1066) [inline pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:7]: Can we add hover text to say what the button does, like the current Refresh and Column buttons have? (the Download button is also missing hover text)

File: pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs [hunk 2]
```diff
@@ -3,6 +3,20 @@ var bsTable;
...
+                text: 'Students Only',
+                icon: 'fa-user-graduate',
+                event: () => {
+                    const table = $('#usersTable');
+                    const filterOn = !table.data('filter-student-only');
+                    table.data('filter-student-only', filterOn);
+
+                    $('.columns button[name=studentsOnly]').toggleClass('active', filterOn);
+                    table.bootstrapTable('filterBy', filterOn ? { role: 'Student' } : {});
```
Comment (mwest1066) [inline pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:16]: It looks like this button defaults to "off" (that is, by default we show all users). Is that correct? I like this default.

File: pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs (hunk 1 above)
Comment (jonatanschroeder) [reply pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:7]: Done.

File: pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs (hunk 2 above)
Comment (jonatanschroeder) [reply pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:16]: Yes, it defaults to show all users.

Comment (jonatanschroeder) [reply pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:16]: To clarify, do you mean "I like the default to stay the way it is" or "I'd like this filter to become on by default"?

Comment (mwest1066) [reply pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs:17]: I like the default to stay the way it is 😄

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**